Gemfile.lock

```diff
@@ -242,6 +266,7 @@ DEPENDENCIES
   factory_girl_rails (~> 4.0)
   figaro
   jquery-rails
+  mailcatcher
   omniauth-facebook
   omniauth-google-oauth2
   pg
```

Comment:
This is a lot of update to the Gemfile.lock. Can you double-check to make sure everything in here is necessary? Also, to save everyone the hassle, avoid updating these gems. Use `bundle install` instead of `bundle update`.